xwiki-commons-tools/xwiki-commons-tool-verification-resources/src/test/java/org/xwiki/tool/checkstyle/UnstableAnnotationCheckTest.java

```diff
@@ -35,15 +32,15 @@
  * @version $Id$
  * @since 8.1M1
  */
-public class UnstableAnnotationCheckTest extends BaseCheckTestSupport
+public class UnstableAnnotationCheckTest extends AbstractModuleTestSupport
 {
     private DefaultConfiguration checkConfig;
 
     @Before
     public void setUp()
     {
         this.checkConfig = createCheckConfig(UnstableAnnotationCheck.class);
-        this.checkConfig.addAttribute("currentVersion", "8.1-SNAPSHOT");
+        this.checkConfig.addAttribute("currentVersion", "8.2-SNAPSHOT");
```

Comment:
@mflorea Can you explain what this version is since it looks like it is so closely related to the Checkstyle version?

@subkrish Please verify the build and run doesn't fail if you revert these snapshot versions in tests. If so, please push the changes requested.